platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/SystemPropertiesAdmin.java

```diff
@@ -218,6 +220,8 @@ public void writeSystemProperties(Map<String, String> updatedSystemProperties) {
             systemDotProperties.put(key, value);
           });
 
+      solrPasswordUpdate(systemDotProperties);
```

Comment:
Ideally `admin-core-impl` should not know about Solr since that is an implementation detail.  Instead admin core could provide an interface to modify system properties before writing them out.  Then you could get a service reference list of those plugins to call here.

Reply:
Thanks Phil. That makes sense. 
@brjeter after reading Phil's comment. I think we we should close this PR and not make these changes to the release branch. Instead, we should have a short design meeting (maybe as part of our daily meeting) to decide how best implement @pklinef 's direction.   

I can work personally with individuals experiencing password synchronization issues to alleviate their problems with configuration changes instead of code changes.

Reply:
@ahoffer and I talked offline. We decided to get this PR merged to 2.13.x, as-is, in order to fix the stability issues for the next 2.13.x release, but we will **not** port this exact change to master due to the design flaw that @pklinef pointed out. On master, we'll implement the better solution.